README.md

```diff
@@ -63,9 +65,11 @@ From TU Graz, Shared RDM Project:
 ## Using the helper script
 
 ~~~bash
-# generate the library
+# Option 1: generate the library with Swagger codegen
 ./helper.sh generate
-# generate from local file: openapi.yaml must be in current dir
+# Option 2: generate using openapi-generator
+GENERATOR_TOOL=openapi ./helper.sh generate
+# Option 3: generate from local file: openapi.yaml must be in current dir
```

Comment:
You're putting option 3 at the same level as option 1 and 2, but it's different.

It's generator1 or generator2, and getting it from master OR local file. That's four conditions. Someone without prior knowledge must be able to read the README and understand the different commands and why they exist. A different command is not an Option, it's a different command.